scripts/remote_replace_cloudbeat.sh

```diff
@@ -0,0 +1,35 @@
+# This script will stop any Cloudbeat process within an agent and will replace the Cloudbeat artifacts with your local built artifacts
+
+# Prerequisites:
+# a. Create the Cloudbeat artifacts using the mage command (for example - "DEV=true PLATFORMS=linux/amd64 SNAPSHOT=true mage -v package")
+# b. The running elastic-agents were built with "DEV=TRUE" flag (https://github.com/elastic/security-team/blob/main/docs/cloud-security-posture-team/Onboarding/cloudbeat-ec2.md#:~:text=Build%20the%20agent%20(We%20are%20building%20with%20DEV%3Dtrue%2C%20so%20we%20can%20load%20cloudbeat.tar.gz%20without%20signature%20verification))
+
+#!/bin/bash
+ARCH=`uname -a | rev | cut -d " " -f 1 | rev` # This should return arm64 on m1 and x86_64 on regular mac
+PREFIX="/usr/share/elastic-agent/data/"
+SUFFIX="/install/cloudbeat-8.4.0-SNAPSHOT-linux-${ARCH}"
+DISTRIBUTION_LOCAL_FOLDER="../build/distributions"
+CLOUDBEAT_IMAGE="cloudbeat-8.4.0-SNAPSHOT-linux-$ARCH"
+
+PODS=`kubectl -n kube-system get pod -l app=elastic-agent -o name`
+for P in $PODS
+do
+    POD=`echo $P | cut -d '/' -f 2`
+    FOLDER=`kubectl -n kube-system exec $POD -- ls $PREFIX`
+    # Kill Cloudbeat process
+    CLOUDBEAT_PID=`kubectl -n kube-system exec "$POD" -- pidof cloudbeat`
+    kubectl -n kube-system exec "$POD" -- kill -9 "${CLOUDBEAT_PID}"
```

Comment:
wouldn't that will trigger a restart of cloudbeat?

Reply:
so maybe we should copy it before killing it?

Reply:
I tried to do it as Amir once suggested on his script, but since we are deleting the artifacts this command cannot be run.

`/usr/share/elastic-agent/data/elastic-agent-bbd93f/install/cloudbeat-8.4.0-SNAPSHOT-linux-x86_64/cloudbeat -E setup.ilm.enabled=false -E setup.template.enabled=false -E management.enabled`

This installation path is the old artifact, and if we would not delete it we will run the agent again and again.
And obviously, if the path is deleted as part of the script we can't run it. 

Upsetting :/

Reply:
> I tried to do it as Amir once suggested on his script

and what was the result? it didn't work?

> This installation path is the old artifact, and if we would not delete it we will run the same old cloudbeat again and again.
And obviously, if the path is deleted as part of the script we can't run it.

but here we are killing it before deleting it so it could trigger a re-run of the old beat. 
I am just suggesting we replace the old beat before killing the current one if that's possible

Reply:
- yeah it didn't work, and didn't work for him as well since there is no artifact folder.
- The thing is, we want the cloudbeat process to be killed, so it will extract the tar.gz and init the process. Currently we don't know how to do the extraction ourselves.
Wondering if it is a good practice to replace running directory while the process still running - I think it can lead to a lot of problems:/

Reply:
Was changed according to our call.

The kill was moved to the end of the script.